package.json

```
@@ -0,0 +1,23 @@
+{
+	"name": "kanban-task-manager",
+	"version": "1.0.0",
+	"description": "App for task management",
+	"scripts": {
+		"test": "echo \"Error: no test specified\" && exit 1"
```

remove also test script, pls
```suggestion
	"scripts": {}
```